### ISSUE TITLE
Book title deleting via edit books causes value error

### ISSUE THREAD

**Issue by OzzieIsaacs** [janeczku/calibre-web#622]
If the book title is deleted an exception is thrown: "ValueError: Filename cannot be empty"
This should be prevented.

**Comment by OzzieIsaacs** [comment]
Remark: In Calibre itself the name is set to unknown in this case

**Comment by vigri** [comment]
We could change that to "unknown" too. Something similar is already implemented for authors:

```
if input_authors == ['']:
    input_authors = [_(u'unknown')]  # prevent empty Author
```

But wouldn't it be better to make the title field mandatory?

**Comment by OzzieIsaacs** [comment]
The Calibre behavior is to set the title to unknown, I‘d prefer to copy the Calibre behavior. I aldready fixed it, and also the #646. But I want to write some tests for it before uploading

**Comment by OzzieIsaacs** [comment]
Fixed
